demos/TicTacToe/Classes/TicTacToeScene.h

```diff
@@ -0,0 +1,14 @@
+#pragma once
+
+#include <array>
+#include <cstdlib>
+#include <iostream>
+
+#include "cocos2d.h"
+
+class TicTacToe : public cocos2d::Layer {
+ public:
+  static cocos2d::Scene* createScene();
```

Comment:
Is this a function that Cocos expects to exist, or did you create it? Unless it's a cocos library function, it should be PascalCase. 

https://google.github.io/styleguide/cppguide.html#Function_Names

Reply:
Some comments around each of these functions, as trivial as they might be, would be nice.

Reply:
My issue is resolved (thanks for adding the comments) but I think you still need ot address Alex's..?

Reply:
Thanks for the reminder. The createScene() is a cocos library function.